cli/sawtooth_cli/identity.py

```diff
@@ -37,6 +37,7 @@
 from sawtooth_cli.protobuf.batch_pb2 import BatchHeader
 from sawtooth_cli.protobuf.batch_pb2 import Batch
 from sawtooth_cli.protobuf.batch_pb2 import BatchList
+from sawtooth_cli.config import _key_to_address
```

Comment:
If this is now a public function of the `sawtooth_cli.config` package, it should have a public version called `setting_key_to_address`, so it's clear what that is referring to once imported.